pkg/logs/message/origin.go

```diff
@@ -61,7 +61,7 @@ func (o *Origin) TagsPayload() []byte {
 
 // TagsToString encodes tags to a single string, in a comma separated format
 func (o *Origin) TagsToString() string {
```

Comment:
This file is a good example of a wrong isolation between different business logics .. I am not very proud of it .. :)
The origin should just be a holder of the tags and should not care about how to present the tags to the different encoders, instead I think the different encoders should have some utilities to format the tags, WDYT ?

Reply:
makes perfect sense 👍

I hoped to have some time to address this before the 7.19 freeze but I don't think I will actually, so I'll merge as-is and address this in a separate PR